package.json

```diff
@@ -61,13 +61,16 @@
     "url": "https://github.com/MarkBind/markbind.git"
   },
   "scripts": {
-    "csslint": "./node_modules/.bin/stylelint \"**/*.css\" \"!**/*.min.css\"",
+    "autolint": "npm run lintfix && npm run csslintfix",
+    "csslint": "./node_modules/.bin/stylelint **/*.css !**/*.min.css",
+    "csslintfix": "./node_modules/.bin/stylelint **/*.css !**/*.min.css --fix",
     "jest": "jest",
     "lint": "./node_modules/.bin/eslint .",
+    "lintfix": "./node_modules/.bin/eslint . --fix",
     "pretest": "npm run lint && npm run csslint",
     "pretestwin": "npm run lint && npm run csslint",
-    "test": "jest && cd test/functional && ./test.sh",
-    "testwin": "jest && cd test/functional && test.bat",
+    "test": "npm run autolint && jest && cd test/functional && ./test.sh",
+    "testwin": "npm run autolint && jest && cd test/functional && test.bat",
```

Comment:
Since linting and checking for errors error in the `pretest` and `pretestwin` scripts, we should run the fixes in those scripts instead.